metadata.rb

```diff
@@ -1,12 +1,13 @@
 name 'kafka-cluster'
 maintainer 'John Bellone'
 maintainer_email 'jbellone@bloomberg.net'
-license 'Apache 2.0'
+license 'Apache-2.0'
 issues_url 'https://github.com/bloomberg/kafka-cookbook/issues'
 source_url 'https://github.com/bloomberg/kafka-cookbook'
 description 'Application cookbook which installs and configures Apache Kafka.'
 long_description 'Application cookbook which installs and configures Apache Kafka.'
 version '1.3.10'
```

Comment:
If we're removing attribute values (as per above) let's bump this up at least to 1.4.

Reply:
Sure